example/sound.cpp

```diff
@@ -0,0 +1,80 @@
+#include <al.h>
+#include <alc.h>
+#define DR_WAV_IMPLEMENTATION
+#include <dr_libs/dr_wav.h>
+
+#include <iostream>
+#include <vector>
+
+int main(int argc, char** argv) {
+	if (argc != 2) {
+		std::cerr << "Syntax: " << argv[0] << " <wav file path>" << std::endl;
+		return -1;
```

Comment:
Small comment: most shells interpret return values as `u8`, and negative / large values are reserved for system errors. Also, Windows GUI apps return 1 on success, so don't use 1 either. 😛